### PR TITLE
Add lavapipe/turnip vulkan drivers

### PR DESCRIPTION
**Pull request by xantares**
https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/8593 was just merged

### CODE REVIEW

**Comment by krh** [comment]
Oh, great, I came here to ask for the exact same thing - thanks @xantares!

File: lib/Parser/Constants.php [hunk 1]
```diff
@@ -70,13 +70,17 @@ abstract class Constants
...
         "anv",
+        "lvp",
         "radv",
+        "tu",
         "v3dv",
     ];
 
     const VK_ALL_DRIVERS_VENDORS = [
         "Intel"     => [ "anv" ],
+        "Lavapipe"  => [ "lvp" ],
```
Comment (MightyCreak) [inline lib/Parser/Constants.php:81]: I'd put `lvp` under the `Software` vendor (as for OpenGL and `llvmpipe`)
```suggestion
        "Software"  => [ "lvp" ],
```

File: lib/Parser/Constants.php [hunk 2]
```diff
@@ -70,13 +70,17 @@ abstract class Constants
...
         "AMD"       => [ "radv" ],
+        "Turnip"    => [ "tu" ],
```
Comment (MightyCreak) [inline lib/Parser/Constants.php:83]: From what I understood, turnip is a driver for Qualcomm hardware, so as for OpenGL, so I'd put `tu` under the `Qualcomm` vendor.
```suggestion
        "Qualcomm"    => [ "tu" ],
```

**Comment by MightyCreak** [comment]
Thank you so much for taking care of this!

**Comment by xantares** [comment]
done

**Comment by MightyCreak** [comment]
Thank you!

**Pull request merged**